### PR TITLE
GitHub docs

### PR DESCRIPTION
**Pull request by Shraddha2104**
Hey!
✨ Added basic instructions for Github node
Please let me know if any changes!

### CODE REVIEW

File: docs/nodes/nodes-library/nodes/GitHub/README.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+- Issue
+	- Create a new issue
+	- Create a new comment on an issue
+	- Edit an issue
+	- Get the data of a single issue
+	- Lock an issue
+
+- Repository
+	- Get the data of a single repository
+	- Return the contents of the repository's license file, if one detected
```
Comment (smamudhan) [inline docs/nodes/nodes-library/nodes/GitHub/README.md:30]: Could you change it to match the node?
image

File: docs/nodes/nodes-library/nodes/GitHub/README.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+	- Returns issues of a repository
+	- Get community profile of a repository with metrics , health score, description, license
```
Comment (smamudhan) [inline docs/nodes/nodes-library/nodes/GitHub/README.md:32]: Could you change this to
```
Get the community profile of a repository with metrics, health score, description, license, ...
```

File: docs/nodes/nodes-library/nodes/GitHub/README.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+	- Get the top 10 popular content paths over the last 14 days
+	- Get the top 10 popular referrering domains over the last 14 days
```
Comment (smamudhan) [inline docs/nodes/nodes-library/nodes/GitHub/README.md:34]: image

File: docs/nodes/nodes-library/nodes/GitHub/README.md [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+
+# GitHub
+
+[GitHub](https://github.com/) provides hosting for software development and version control using Git. It offers the distributed version control and source code management (SCM) functionality of Git, access control and several collaboration features such as bug tracking, feature requests, task management, and wikis for every project.
+
+::: tip 🔑 Credentials
+You can find authentication information for this node [here](../../../credentials/Github/README.md).
+:::
+
+## Basic Operations
```
Comment (smamudhan) [inline docs/nodes/nodes-library/nodes/GitHub/README.md:13]: Can you make sure the formatting matches that of other nodes' documentation?

File: docs/nodes/nodes-library/nodes/GitHub/README.md [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+![A workflow with the GitHub node](./workflow.png)
+
+### 1. Start node
+
+The start node exists by default when you create a new workflow.
+
+### 2. GitHub node
+
+1. First of all, you'll have to enter credentials for the GitHub node. You can find out how to do that [here](../../../credentials/Github/README.md).
+2. Select the *Repository* option under the *Resource* field.
```
Comment (smamudhan) [inline docs/nodes/nodes-library/nodes/GitHub/README.md:59]: Please change `*Repository*` to `'Repository'`

File: docs/nodes/nodes-library/nodes/GitHub/README.md [hunk 6]
```diff
@@ -0,0 +1,63 @@
...
+3. Select the *Get Profile* option under the *Operation* field.
```
Comment (smamudhan) [inline docs/nodes/nodes-library/nodes/GitHub/README.md:60]: Please change `*Get Profile*` to `'Get Profile'`

File: docs/nodes/nodes-library/nodes/GitHub/README.md [hunk 7]
```diff
@@ -0,0 +1,63 @@
...
+4. Enter the repository owner in the *Repository Owner* field.
+5. Enter the repository name in the *Repository Name* field.
+6. Click on *Execute Node* to run the workflow.
```
Comment (smamudhan) [inline docs/nodes/nodes-library/nodes/GitHub/README.md:63]: Could you end all your files with a newline?

**Comment by Shraddha2104** [comment]
Hey @smamudhan,
Updated according to your feedback.
Let me know if any changes are needed

**Comment by tanay1337** [comment]
Merged, thanks a lot @Shraddha2104 🎉

**Pull request merged**